docs/source/conf.py

```diff
@@ -66,6 +65,7 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 
 html_static_path = ["_static"]
+html_css_files = [] # avoid undefined errors
```

Comment:
ci job is passing without this change after #1333 ... Is this change here actually needed?